app/parser.py

```diff
@@ -0,0 +1,35 @@
+from app.database import get_database
+import click
+
+# parses a .csv that is passed to it.  Creates an array with
+# the data
+def parse(file):
+    file_contents_by_row = []
+    last_line = None
+    with open(file) as sample_file:
+        for line in sample_file:
+            if not last_line:
```

Comment:
What would happen if there was a blank line in the file? Would it see it as none and assume it was the last_line in the file? Is this check necessary? I think python knows how to sense ends of files inherently, but I could be wrong.

Reply:
There's probably a way to do it. I'll look into it.